AprilTagsVision/robot.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+
+import wpilib
+import wpilib.cameraserver
+
+
+"""
```

Comment:
This isn't a docstring – it's not the first statement of a module/class/function.